src/pages/editOverview.test.js

```diff
@@ -0,0 +1,179 @@
+import '@testing-library/jest-dom'
+import { configure, render, within } from '@testing-library/react'
+import React from 'react'
+import { BrowserRouter } from 'react-router-dom'
+import { AppContext, UserContext } from '../context/index.js'
+import { testDataExchange, testUserContext } from '../utils/builders.js'
+import { EditPage } from './editOverview.js'
+
+jest.mock('@dhis2/ui', () => {
+    const ui = jest.requireActual('@dhis2/ui')
+
+    return {
+        __esModule: true,
+        ...ui,
+        SharingDialog: () => <div data-test="mock-sharing-dialog" />,
+    }
+})
+
+const setUp = (
+    ui,
+    {
+        aggregateDataExchanges = [testDataExchange(), testDataExchange()],
+        userContext = testUserContext(),
+    } = {}
+) => {
+    const originalWarn = console.warn
+    jest.spyOn(console, 'warn').mockImplementation((value) => {
+        if (!value.match(/No server timezone/)) {
```

Comment:
You might also be able to get rid of this warning by specifying the time zone as a jest variable? (e.g. by modifying the test script to have time zone defined, e.g. https://github.com/dhis2/aggregate-data-entry-app/blob/master/package.json#L11). I think not having the time zone should not affect the tests here in any case.

Reply:
I've tried this but can not get it to work. I'll message you bout it